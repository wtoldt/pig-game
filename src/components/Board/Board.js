import React from 'react';
import './Board.scss';

export default function Board ({children}) {

  return (
    <div className="pg-board">
      {children}
    </div>
  );
}