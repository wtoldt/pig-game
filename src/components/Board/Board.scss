@import '../../styles/consts.scss';

.pg-board {
  display: flex;
  height: 500px;
  width: 1000px;
  border: solid 2px black;
  border-radius: $big-panel-br;
}