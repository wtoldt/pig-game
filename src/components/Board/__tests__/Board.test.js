import { render } from '@testing-library/react';
import Board from '../Board';

const defaultProps = {};

function getWrapper(props={}, children = 'foo') {
  return render(
    <Board {...defaultProps} {...props}>
      {children}
    </Board>
  );
}

describe('Board Component', () => {

  describe('Snapshot testing', () => {
    it('Default snapshot', () => {
      const { container } = getWrapper();
      expect(container).toMatchSnapshot();
    });
  });
});
