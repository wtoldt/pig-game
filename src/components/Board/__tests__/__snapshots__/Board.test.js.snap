// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Board Component Snapshot testing Default snapshot 1`] = `
<div>
  <div
    class="pg-board"
  >
    foo
  </div>
</div>
`;
